pkg/execution/state/redis_state/queue.go

```diff
@@ -6,37 +6,38 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/inngest/inngest/pkg/telemetry/redis_telemetry"
 	"math"
 	"strconv"
 	"strings"
 	"sync"
 	"time"
 	"unsafe"
 
-	"golang.org/x/sync/semaphore"
-
 	"github.com/VividCortex/ewma"
 	"github.com/cespare/xxhash/v2"
 	"github.com/google/uuid"
 	"github.com/hashicorp/go-multierror"
+	"github.com/jonboulle/clockwork"
+	"github.com/oklog/ulid/v2"
+	"github.com/redis/rueidis"
+	"github.com/rs/zerolog"
+	"golang.org/x/sync/semaphore"
+	"gonum.org/v1/gonum/stat/sampleuv"
+	"lukechampine.com/frand"
+
 	"github.com/inngest/inngest/pkg/backoff"
 	"github.com/inngest/inngest/pkg/consts"
 	osqueue "github.com/inngest/inngest/pkg/execution/queue"
 	"github.com/inngest/inngest/pkg/execution/state"
 	"github.com/inngest/inngest/pkg/logger"
 	"github.com/inngest/inngest/pkg/telemetry"
+	"github.com/inngest/inngest/pkg/telemetry/redis_telemetry"
 	"github.com/inngest/inngest/pkg/util"
-	"github.com/oklog/ulid/v2"
-	"github.com/redis/rueidis"
-	"github.com/rs/zerolog"
-	"gonum.org/v1/gonum/stat/sampleuv"
-	"lukechampine.com/frand"
 )
 
-var (
-	PartitionSelectionMax int64 = 100
-	PartitionPeekMax      int64 = PartitionSelectionMax * 3
+const (
```

Comment:
Unrelated change; these never seem to be modified (and it's unclear that they could be modified safely at runtime).